docs/src/quickstart/index.md

```diff
@@ -57,356 +70,184 @@ for these executables in your `$PATH`.
-            Some(mut g) => {
-                g.last_seen = block.height;
-                g
-            }
-            None => Greeter {
-                id: greeter_id,
-                first_seen: block.height,
-                name: event.person.name.to_string(),
-                last_seen: block.height,
-            },
-        };
-
-        // Both entity saves will occur in the same transaction
-        greeting.save();
-        greeter.save();
-    }
-}
-```
+### 2.3 Deploying our index
 
-### 5. Compile the index
+By now we have a brand new index that will index some blocks and transactions, but now we need to build and deploy it in order to see it in action.
 
-```bash
-cd indexer/hello-index
+#### 2.3.1 Starting an indexer service
 
-forc index build --manifest hello_index.manifest.yaml
-```
+- To start an indexer service, we'll be spinning up Postgres and Fuel indexer containers via docker compose. Our indexer service will connect to Fuel's `beta-2` network so that we can index blocks and transactions from an _actual_ Fuel node. We'll use the `docker compose` file below, and spinning everything up with `docker compose up`.
 
-> IMPORTANT: As of this writing, there is a small bug in newly built Fuel indexer WASM modules that produces a WASM runtime error due an errant upstream dependency. For now, a quick workaround requires using `wasm-snip` to remove the errant symbols from the WASM module. More info can be found in the related script [here](https://github.com/FuelLabs/fuel-indexer/blob/master/scripts/stripper.bash).
->
+> IMPORTANT: Ensure that any local Postgres instance that is running on port `5432` is stopped.
 >
-> IMPORTANT: Be sure to add your new WASM module to your index manifest as shown below.
-
-```yaml
-namespace: fuel_examples
-identifier: hello_index
-abi: examples/hello-world/contracts/greeting/out/debug/greeting-abi.json
-start_block: 1
-graphql_schema: examples/hello-world/schema/hello_world.schema.graphql
-module:
-  wasm: target/wasm32-unknown-unknown/release/hello_index.wasm
+> You can open up a `docker-compose.yaml` file in the same directory as your index project, and paste the YAML content below to this file.
+
+```text
+version: "3.9"
+services:
+  postgres:
+    image: postgres:latest
+    ports:
+      - "5432:5432"
+    volumes:
+      - .:/usr/local/postgres
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - PGUSER=postgres
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready", "-U", "postgres", "-d", "postgres"]
+      interval: 30s
+      timeout: 60s
+      retries: 5
+      start_period: 80s
+  fuel-indexer:
+    image: ghcr.io/fuellabs/fuel-indexer:latest
+    command: bash -c "sleep 2 && ./fuel-indexer --fuel-node-host node-beta-2.fuel.network --fuel-node-port 80 --postgres-host postgres --postgres-password postgres --graphql-api-host 0.0.0.0"
+    ports:
+      - "29987:29987"
+    volumes:
+      - .:/usr/local/fuel-indexer
+    depends_on:
+      - postgres
```

Comment:
I had postgres running in the background, 
```
harry@xps:~/dev/test/quick_start_compose$ sudo ss -lptn 'sport = :5432'
State                    Recv-Q                   Send-Q                                      Local Address:Port                                       Peer Address:Port                   Process                   
LISTEN                   0                        244                                             127.0.0.1:5432                                            0.0.0.0:*                       users:(("postgres",pid=1584,fd=5))
```
so kill your local postgres because its going to try to bind on that default port
```
sudo systemctl stop postgresql

docker compose up
```
And then it will stand up